platform/ios/src/MGLMapView.mm

```diff
@@ -750,6 +753,20 @@ - (void)glkView:(__unused GLKView *)view drawInRect:(__unused CGRect)rect
         _mbglMap->render();
 
         [self updateUserLocationAnnotationView];
+        
+        // Hide all annotation views
+        for (MGLAnnotationView *annotationView in _annotationViews) {
+            annotationView.hidden = YES;
+        }
+        
+        // Update visible annotation views
+        std::vector<MGLAnnotationTag> annotationTags = [self annotationTagsInRect:CGRectInset(self.bounds, 0, 0)];
+        for (std::size_t i = 0; i != annotationTags.size(); ++i) {
+            MGLAnnotationContext &annotationContext = _annotationContextsByAnnotationTag[annotationTags[i]];
+            CGPoint annotationPoint = [self convertCoordinate:annotationContext.annotation.coordinate toPointToView:self.glView];
```

Comment:
Nit: We tend to use `self` as the reference view when calling the conversion methods. This is normally OK, since `glView` is constrained to fill the map view, but there are cases like #4352 where the bounds won’t quite match. We should be consistent in the meantime.